src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-sentry_sdk.py

```diff
@@ -18,3 +19,9 @@
                  "sentry_sdk.integrations.argv",
                  "sentry_sdk.integrations.logging",
                  "sentry_sdk.integrations.threading"]
+
+if hasattr(si, '_AUTO_ENABLING_INTEGRATIONS'):
```

Comment:
Can we have a comment saying exactly what type/format `si._AUTO_ENABLING_INTEGRATIONS` is? I'm guessing list of string names of integrations where the names exclude the `sentry_sdk.integrations.` prefix?

Reply:
Yes, that's the list https://github.com/getsentry/sentry-python/blob/c6b6f2086b58ffc674df5c25a600b8a615079fb5/sentry_sdk/integrations/__init__.py#L54-L66

Reply:
Can you put that url into the code.